.circleci/docker/Dockerfile

```diff
@@ -34,11 +34,6 @@ ENV CUDA_PATH /usr/local/cuda
 ENV CC "${cc}"
 ENV CXX "${cxx}"
 
-# Whether to build torch and torch_xla libraries with CXX ABI
-ENV _GLIBCXX_USE_CXX11_ABI "${cxx_abi}"
```

Comment:
Dockerfile change needs to be coordinated with the upstream, when are we planning to merge this PR? Can we aim for early next week?

Reply:
Sounds good, we can do it after the pin update whenever it makes most sense for the team.

Reply:
Is this still a valid concern?